src/qt/rpcconsole.cpp

```diff
@@ -843,8 +865,18 @@ void RPCConsole::on_lineEdit_returnPressed()
 
         cmdBeforeBrowsing = QString();
 
+        void *ppwallet_v = nullptr;
+#ifdef ENABLE_WALLET
+        const int wallet_index = ui->WalletSelector->currentIndex();
+        if (wallet_index > 0) {
+            CWalletRef *ppwallet = (CWalletRef*)ui->WalletSelector->itemData(wallet_index).value<void*>();
+            ppwallet = new CWalletRef(*ppwallet);  // Refcount
```

Comment:
In commit "Qt: Add wallet selector to debug console"

What does refcount comment mean? Is this a todo suggesting that you want to add refcounting in the future? Should clarify comment or maybe remove it.

Reply:
Clarified.